src/Components/index.tsx

```diff
@@ -0,0 +1,8 @@
+import Button from "./Button/Button";
```

Comment:
Maintaining this file will be too complex in future. Its better to avoid this

We will end up adding all components here atoms, molecules etc. Something which we want to avoid